src/controllers/api.controller.ts

```diff
@@ -127,47 +130,47 @@ export const claimsGet = async (
       skip: (Number(page) - 1) * Number(limit),
       take: 10,
       orderBy: {
-        id: 'desc',
+        id: "desc",
       },
       include: {
         edgesFrom: {
           skip: (Number(page) - 1) * Number(limit),
           take: Number(limit) ? Number(limit) : undefined,
-          select :{
+          select: {
             id: true,
             claimId: true,
-            startNodeId:true,
-            endNodeId:true,
-            label:true,
-            thumbnail:true,
-            claim :true,
-            endNode:true,
```

Comment:
this is hard to review because of the linter changes.  please separate those and you can merge linter changes without review